test/test_ops.py

```diff
@@ -1688,12 +1785,40 @@ class TestRefsOpsInfo(TestCase):
 
     @parametrize("op", ref_ops_names)
     def test_refs_are_in_python_ref_db(self, op):
+        err = f"{op} does not have an entry in python_ref_db"
         if op in self.skip_ref_ops:
-            raise unittest.SkipTest(f"{op} does not have an entry in python_ref_db")
-        self.assertIn(op, self.ref_db_names)
+            raise unittest.SkipTest(err)
+        # skip aliases since we don't want them defined in the ref db
+        if op in self.ref_alias_names:
+            return
+        self.assertIn(op, self.ref_db_names, msg=err)
+
+    # like 'test_refs_are_in_python_ref_db' but check in the other direction,
+    # this includes aliases since those are not allowed in 'ref_db_names'
+    @parametrize("op", itertools.chain(ref_db_names, ref_alias_names))
+    def test_python_refs_are_in_ops_names(self, op):
+        # TODO: skip nvfuser refs since those don't have __all__ and it's not
+        # clear at the moment what would be the best way to check those
+        # https://github.com/pytorch/pytorch/issues/85081
+        if op.startswith("ops.nvprims"):
+            return
+        self.assertIn(op, self.ref_ops_names)
+
+    @parametrize("op", ref_db_names)
```

Comment:
I don't think I'd parametrize this test -- it's just asserting a property of the entire sequence. It can fail and the message can describe which aliases are redundantly OpInfo'd

Reply:
Why not? I feel like with the decorator it's more clear which part of the test is actually important. There's less boilerplate and it doesn't take that much time to run. The other tests in the class also use the decorator, so it's all consistent.

Reply:
Multiply the "it doesn't take that much time to run" by the number of CI jobs times the number of commits time the number of PRs and...
I agree with Mike on this one. A better way of doing this is working with the whole set and report all the errors at once in one test.